nixos/roles/mysql.nix

```diff
@@ -145,6 +159,12 @@ with builtins;
         sysdate-is-now             = 1
         sql_mode                   = NO_ENGINE_SUBSTITUTION
 
+        log_slow_verbosity = 'full'
+        slow_query_log = ON
+        long_query_time = 100
```

Comment:
This means 100 seconds.

Also, we should `log_slow_admin_statements` and `log_queries_not_using_indexes`.